packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -364,11 +394,46 @@ export default class TabAdvanced extends Component {
     const isEnterprisePlan = getXoaPlan().value >= ENTERPRISE.value
     const isMigrationCompressionAvailable = pool.migrationCompression !== undefined
 
+    const dcScopeIp = this._getDcScopeIp()
+    const dcNetScopeIp = this._getDcNetScopeIp()
+
     return (
       <div>
         <Container>
           <Row>
             <Col className='text-xs-right'>
+              {dcScopeIp === undefined ? (
+                <TabButton
+                  btnStyle='warning'
+                  icon='deploy'
+                  handler={() => {}}
+                  labelId='deployDcScope'
+                  redirectOnSuccess='/hub/recipes'
+                />
+              ) : (
+                <TabButton
+                  btnStyle='success'
+                  handler={() => window.open(`http://${dcScopeIp}`)}
```

Comment:
If I remember correctly, DC instance (scope/netScope) is always on the `https` protocol.
@pierrebrunet289 can you confirm?